data/data/openstack/topology/private-network.tf

```diff
@@ -128,15 +129,15 @@ resource "openstack_networking_floatingip_associate_v2" "api_fip" {
 }
```

Comment:
We should also probably update comment at https://github.com/openshift/installer/pull/3822/files/d19d119568dbb84e6ada4a7289c34c4953defed8#diff-2e9ee288606583af9d7aff21a6a2cf81R110-R123